feature/policy_forwarding/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,174 @@
+# PF-1.6: Policy forwarding based GUE tunnel decap to default and non-default network-instance
+
+## Summary
+This test ensures NOS is able to host multiple VRFs, perform GUE tunnel DECAP in the default VRF and also allows for gradual traffic migration from Default to Non-Default VRF using VRF selection policy.
+
+
+## Test environment setup
+
+### Topology
+Create the following connections:
+```mermaid
+graph LR; 
+subgraph ATE1 [ATE1]
+    A1[Port1] 
+end
+subgraph DUT1 [DUT1]
+    B1[Port1]
+    B2[Port2]
+end
+subgraph ATE2 [ATE2]
+    C1[Port1] 
+end
+A1 <-- IBGP(ASN100) --> B1; 
+B2 <-- EBGP(ASN100:ASN200) --> C1;
+```
+
+### Advertisements:
+
+	ATE1:Port1 advertises following prefixes over IBGP to DUT:Port1
+		- IPv4Prefix1/24 IPv6Prefix1/64
+		- IPv4Prefix2/24 IPv6Prefix2/64
+		- IPv4Prefix3/24 IPv6Prefix3/64
+		- IPv4Prefix4/24 IPv6Prefix4/64
+		- IPv4Prefix5/24 IPv6Prefix5/64
+
+	ATE2:Port1 advertieses following prefixes to DUT:Port2 over EBGP
+		- IPv4Prefix6/24 IPv6Prefix6/64
+		- IPv4Prefix7/24 IPv6Prefix7/64
+		- IPv4Prefix8/24 IPv6Prefix8/64
+		- IPv4Prefix9/24 IPv6Prefix9/64
+		- IPv4Prefix10/24 IPv6Prefix10/64
+
+### Flows:
+
+	From ATE1:Port1 to ATE2:Port1
+		- IPv4Prefix1/24 to IPv4Prefix6/24 
+		- IPv6Prefix1/64 to IPv6Prefix6/64 
+
+		- IPv4Prefix2/24 to IPv4Prefix7/24 
+		- IPv6Prefix2/64 to IPv6Prefix7/64 
+
+		- IPv4Prefix3/24 to IPv4Prefix8/24 
+		- IPv6Prefix3/64 to IPv6Prefix8/64 
+
+		- IPv4Prefix4/24 to IPv4Prefix9/24 
+		- IPv6Prefix4/64 to IPv6Prefix9/64 
+
+		- IPv4Prefix5/24 to IPv4Prefix10/24 
+		- IPv6Prefix5/64 to IPv6Prefix10/64 
+
+	Flows from ATE2:Port1 to ATE1:Port1
+		- IPv4Prefix6/24 to IPv4Prefix1/24 
+		- IPv6Prefix6/64 to IPv6Prefix1/64 
+
+		- IPv4Prefix7/24 to IPv4Prefix2/24 
+		- IPv6Prefix7/64 to IPv6Prefix2/64 
+
+		- IPv4Prefix8/24 to IPv4Prefix3/24
+		- IPv6Prefix8/64 to IPv6Prefix3/64 
+
+		- IPv4Prefix9/24 to IPv4Prefix4/24 
+		- IPv6Prefix9/64 to IPv6Prefix4/64
+
+		- IPv4Prefix10/24 to IPv4Prefix5/24 
+		- IPv6Prefix10/64 to IPv6Prefix5/64 
+
+
```

Comment:
You need to specify a DUT configuration.  The configuration should show the network-instance (vrf) you require, along with any policy forwarding rules.  The configuration can be described in English and must also be specified in OpenConfig JSON.   For example, see https://github.com/openconfig/featureprofiles/pull/3383/files

I can help you with the JSON offline if you first describe what you need in English

The whole configuration we want to test should be here.  We'll refer to this as  the config generation part.   Generating the config is environment set up, and not part of the DUT testing.

Then in the test procedure, you can use gnmi.Set REPLACE to push the config to the DUT.  Pushing the earlier generated config to the DUT belongs in the test procedure, because we expect it might not work. :)

Reply:
Done , added in english , PTAL